common/types/block.go

```diff
@@ -50,7 +50,7 @@ func getLayersPerEpoch() int32 {
 // SetLayersPerEpoch sets global parameter of layers per epoch, all conversion from layer to epoch use this param
 func SetLayersPerEpoch(l int32) {
 	atomic.StoreInt32(&layersPerEpoch, l)
-	atomic.StoreInt32(&EffectiveGenesis, getLayersPerEpoch()*2-1)
+	atomic.StoreInt32(&EffectiveGenesis, l*2-1)
```

Comment:
I can't visually distinguish between "el times two minus one" or "one times two minus one" here - can we rename the `l` variable?